src/scripts/main.js

```diff
@@ -96,57 +98,46 @@ const addButton = document.querySelector('form>button');
 addButton.addEventListener('click', (e) => {
   e.preventDefault();
 
-  let isValid = true;
-
   if (nameInput.value.length < 4) {
-    pushNotification(
+    return pushNotification(
       150,
       10,
       'Error message',
       'Name value is too short',
       'error',
     );
-    isValid = false;
   }
 
-  if (+ageInput.value < 18 || +ageInput.value > 90) {
-    pushNotification(
+  if (positionInput.value.length < 3 || positionInput.value === 0) {
```

Comment:
I still can add spaces only:

![image](https://github.com/user-attachments/assets/b1ce648f-f649-489e-864f-a6cdb608a307)

use `.trim()` method and validate the resulting value

Also, make Office (actually all fields) required